### DISCUSSION TITLE
Popover Date Formatting

### DISCUSSION THREAD

**Discussion by roysim11** [MMRIZE/MMM-CalendarExt3#189] [Q&A] [answered]
I asked this under a previous post regarding popevers, but I'm not sure if this got lost and I'm sorry for the duplication.  Popovers work great for me for both the day's total events and then the individual events. However, the date formats are different. The day's total events display correctly, which would be en-US (mm/dd/yr) 12h AM/PM. The format for the individual event popovers is (dd/mm/yr) 24hr. Is there a config line to get the individual event format to match the total events format?

**Comment by eouia** [comment]
There is an option `popeverPeriodOptions`, and the default value is
```js
{
  timeStyle: 'short',
  dateStyle: 'long',
}
```
Under `'en-US'` locale, It should ordinarily show `mm/dd/yy` and `hh:mm AM`

<img width="601" alt="image" src="https://github.com/user-attachments/assets/59380077-8b44-4114-8de6-daaead88c0a7">

...
But, it follows only the MM's locale not this module's locale. That is not intended bug. I'll fix it soon to respect the module's locale config.
Before the fix, you can set your MM's locale as `en-US`. (I guess you may have set it as another locale)

**Reply by roysim11** [reply]
I have the locale set as `en-US` in both the main MM config and within the calendarEXT3 module.  I have also added the locale into the default calendar just as an experiment and still have the same issue.  

Just for clarification, one type of popover displays correctly and the other doesn't, see the following:

**Summary of full day:**
![Cal Snip1](https://github.com/user-attachments/assets/06f5f3cc-14f2-4247-9467-84740bc2cc5a)

**Single event popover:**
![Cal Snip2](https://github.com/user-attachments/assets/a0728ab7-a1ec-4aa4-98a6-05258ae31741)

**Reply by eouia** [reply]
Wait a while. As I said, I'm fixing it.

**Reply by eouia** [reply, marked as answer]
I fixed. Please update and try it.

**Reply by roysim11** [reply]
Sorry, didn't mean to sound like I was rushing you, just making sure my issue was clear.  An issue which is fixed!  Recent update worked like a charm.  Thank you!
